commands/output/output.go

```diff
@@ -78,10 +88,29 @@ func getStatusString(r *review.Summary) string {
 }
 
 // PrintSummary prints a single-line summary of a review.
-func PrintSummary(r *review.Summary) {
+func PrintSummary(r *review.Summary, repo *repository.Repo) {
+	var use_color bool = false
+
+	if repo != nil {
+		use_color = (*repo).GetColorBool("color.appraise")
+	}
+
 	statusString := getStatusString(r)
 	indentedDescription := strings.Replace(r.Request.Description, "\n", "\n  ", -1)
-	fmt.Printf(reviewSummaryTemplate, statusString, r.Revision, indentedDescription)
+
```

Comment:
We try to reduce the amount of nesting for code blocks by short-circuiting when possible.

In this case, we can check `if !useColor` right here, and then return `fmt.Printf(reviewSummaryTemplate, statusString, r.Revision, indentedDescription)`.

i.e.

```go
	if !useColor {
		fmt.Printf(reviewSummaryTemplate, statusString, r.Revision, indentedDescription)
		return
	}
```